src/_kw

```diff
@@ -22,4 +24,17 @@ _kw() {
   fi
 }
 
+_kw_build() {
```

Comment:
Any specific reason for keeping the `{` in this line instead of a new line? Check: https://kworkflow.org/content/codingstyle.html#functions